airbyte-cdk/python/README.md

```diff
@@ -152,8 +159,8 @@ HTTP requests to `localhost:8113/data` should now return the body defined in the
 
 ## Coming Soon
 
-* Full OAuth 2.0 support \(including refresh token issuing flow via UI or CLI\)
-* Airbyte Java HTTP CDK
-* CDK for Async HTTP endpoints \(request-poll-wait style endpoints\)
-* CDK for other protocols
-* Don't see a feature you need? [Create an issue and let us know how we can help!](https://github.com/airbytehq/airbyte/issues/new?assignees=&labels=type%2Fenhancement&template=feature-request.md&title=)
+- Full OAuth 2.0 support \(including refresh token issuing flow via UI or CLI\)
```

Comment:
This is sus — I think maybe some of these are already implemented and not updated in docs? /cc @evantahler

Reply:
👍 on removing this.

Async is in-the works, and talking about Java doesn't make sense here. We've made some connectors for "other protocols" (SOAP), but I don't think folks really care about the internals.